packages/pf4-component-mapper/src/files/wizard.d.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactNode } from "react";
+import { Field } from "@data-driven-forms/react-form-renderer";
+
+export interface WizardButtonLabels {
+  submit?: ReactNode;
+  cancel?: ReactNode;
+  back?: ReactNode;
+  next?: ReactNode;
+}
+
+export interface WizardProps {
+  buttonLabels?: WizardButtonLabels;
+  buttonsClassName?: string;
+  title?: ReactNode;
+  description?: ReactNode;
+  isCompactNav?: boolean;
+  inModal?: boolean;
+  setFullWidth?: boolean;
+  setFullHeight?: boolean;
+  isDynamic?: boolean;
+  showTitles?: boolean;
+  crossroads?: string[];
+  fields: Field[];
```

Comment:
It isn't a field, it shoul be something like WizardStep

```ts
WizardStep  {
   name: String;
   nextStep: String | Object | Function;
   substepOf: String;
   fields: Field[];
   ... (please see https://data-driven-forms.org/component-example/wizard?mapper=pf4 # Docs for steps )
}
```

Reply:
yeah name is required always in `Field` type